src/program/snabbvmx/lwaftr/lwaftr.lua

```diff
@@ -84,6 +84,16 @@ function parse_args (args)
    return opts, conf_file, id, pci, mac, sock_path, mirror_id
 end
 
+local function dirname (filename)
```

Comment:
There's a `dirname` function in `src/core/lib.lua` and another one in `src/syscall/util.lua`, maybe one of those may be used here?